src/modules/generators/DataFuzzer.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2021 European Spallation Source, ERIC. See LICENSE file
+//===----------------------------------------------------------------------===//
+///
+/// \file
+///
+/// \brief Class for fuzzing a data buffer: writing random data at random
+/// locations according to specifications.
+///
+/// Meant for use in data generators
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <random>
+
+class DataFuzzer {
+public:
+
+  DataFuzzer() {
+    std::seed_seq SeedSequence{1, 2, 3, 4, 5};
+    Generator = new std::mt19937(SeedSequence);
```

Comment:
You have a memory leak here: I suggest that you do not use `new` at all here but instead put the following in the `private` section of your class declaration:

```c++
std::seed_seq SeedSequence{1, 2, 3, 4, 5};
std::mt19937 Generator{SeedSequence};
```